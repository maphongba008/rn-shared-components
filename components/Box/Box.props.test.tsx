import React from "react";
import { Text } from "react-native";
import renderer from "react-test-renderer";
import { Box } from "./index";
describe("test Box", () => {
  it("render correctly", () => {
    const tree = renderer
      .create(
        <Box
          mBorderBottom
          mBackgroundColor="red"
          mBorderRadius
          mCenter
          mRow
          mFull
        >
          <Text>Centered text</Text>
        </Box>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
