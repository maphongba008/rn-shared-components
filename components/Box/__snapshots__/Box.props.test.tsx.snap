// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Box render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "red",
        "borderBottomWidth": 1,
        "borderRadius": 4,
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      },
      undefined,
    ]
  }
>
  <Text>
    Centered text
  </Text>
</View>
`;
